### PR TITLE
[Timeseries][CI]Refactor CI to skip AutoMM and Tabular tests w.r.t timeseries changes

### PR DESCRIPTION
**Pull request by prateekdesai04**
*Description of changes:*
Skip `test_multimodal_*` and `test_tabular_*` whenever any changes related to Timeseries is done.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -179,8 +183,10 @@ jobs:
           filters: |
             other_than_docs:
               - '!(docs/**)**'
+            exclude_timeseries:
```
Comment (canerturkmen) [inline .github/workflows/continuous_integration.yml:186]: nit: should it be `other_than_timeseries` for consistency?

**Review by canerturkmen — COMMENTED** [review]
One small comment. LGTM otherwise.

**Review by Innixma — APPROVED** [review]
LGTM!

File: .github/workflows/continuous_integration.yml [hunk 2]
```diff
@@ -199,8 +205,10 @@ jobs:
           filters: |
             other_than_docs:
               - '!(docs/**)**'
+            other_than_timeseries:
+              - '!(timeseries/**)**'
       - name: Test Multimodal Others on AWS Batch
-        if: steps.changes.outputs.other_than_docs == 'true'
+        if: steps.changes.outputs.other_than_docs == 'true' && steps.changes.outputs.exclude_timeseries == 'true'
```
Comment (Innixma) [inline .github/workflows/continuous_integration.yml:211]: Needs to be `steps.changes.outputs.other_than_timeseries`, ditto for other instances

Comment (prateekdesai04) [reply .github/workflows/continuous_integration.yml]: made the changes, sorry for the oversight

**Review by canerturkmen — APPROVED** [review]

**Pull request merged**